File: src/main/java/xyz/thewhitedog9487/WebAPI/WebApiApplication.java
package xyz.thewhitedog9487.WebAPI;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import xyz.thewhitedog9487.WebAPI.Startup.SystemdInstallerManager;

@SpringBootApplication
public class WebApiApplication {
	public static void main(String[] args) {
		//TODO: 从配置文件或环境变量读取并设置HTTP端口号
        var SpringApp = new SpringApplication(WebApiApplication.class);
        SystemdInstallerManager.ProcessArguments(args);
		SpringApp.run(args);
	}

}

File: src/main/kotlin/xyz/thewhitedog9487/WebAPI/WebApiApplication.kt
package xyz.thewhitedog9487.WebAPI

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.security.core.context.SecurityContextHolder
import reactor.core.publisher.Hooks
import xyz.thewhitedog9487.WebAPI.Startup.ProcessArguments

@SpringBootApplication
class WebApiApplication

fun main(args: Array<String>) {
	Hooks.enableAutomaticContextPropagation()
	SecurityContextHolder.setStrategyName(SecurityContextHolder.MODE_INHERITABLETHREADLOCAL)
	ProcessArguments(args)
	runApplication<WebApiApplication>(*args)
}